src/common/models/Projection.ts

```diff
@@ -132,6 +132,9 @@ export class Projection {
   // Test Positive series as values between 0-1.
   private readonly testPositiveRate: Array<number | null>;
   private readonly contractTracers: Array<number | null>;
+  private readonly caseDensityCases: Array<number | null>;
+  private readonly caseDensityDeaths: Array<number | null>;
```

Comment:
nit: Maybe call these caseDenistyByCases and caseDensityByDeaths since they're the same metric, just calculated by two different mechanisms.

Also, because we want these to be usable via the getDataset() method which takes a string and expects a field with that name to exist on the class, we need to add these names to `DatasetId` above.